tests/gtest/datahandle_metadata_test.cpp

```diff
@@ -161,4 +163,150 @@ TEST_F(DatahandleMetadataTest, Metadata_Different_Size_Double) {
     EXPECT_EQ(metadata["axis"], different_size_metadata["axis"]);
 }
 
+TEST_F(DatahandleMetadataTest, Metadata_One_crossLine_Single) {
+
+    // const std::string SINGLE_XLINE_DATA = "file://10_single_xline.vds";
```

Comment:
:broom:  ? Or  on the opposite, remove it from the named files to make consistent with tests below?